src/store/costModels/actions.ts

```diff
@@ -126,3 +128,32 @@ export const deleteCostModel = (
       });
   };
 };
+
+export const redirectToCostModelFromSourceUuid = (source_uuid: string, history: H.History) => {
+  return (dispatch: Dispatch) => {
+    return apiGetCostModels(`source_uuid=${source_uuid}`).then(res => {
+      if(res.data.data.length === 0) {
+        dispatch(
+          addNotification({
+            title: i18next.t('cost_models_router.error_title'),
+            description: i18next.t('cost_models_router.no_match'),
+            variant: 'danger',
+            dismissable: true,
+          })
```

Comment:
I don't understand why we need to dispatch a 'danger' notifications? 

Couldn't we disable the 'View all price lists' option in the kebab? The /reports/openshift/costs API already provides the `source_uuid` in the report. This information is already available to each row of the table.

<img width="541" alt="Screen Shot 2020-08-05 at 4 10 53 PM" src="https://user-images.githubusercontent.com/17481322/89459214-4a1f6a00-d736-11ea-85fe-1b9c4f42b541.png">

We should be able to perform a `history.push` via the kebab's `onClick` event.

Reply:
Done. :+1:

Reply:
Although, we do need this redux thunk because we need to get the cost model that this source uuid is asigned to.